bin/configure

```diff
@@ -56,8 +55,10 @@ update_asdf_ruby_gems() (
 )
 
 install_codez_python_deps() (
+  # necessary because of https://github.com/pallets/markupsafe/issues/57#issuecomment-597105876
+  python3.9 -m pip install --upgrade pip setuptools wheel
   pushd "$OPENAPI_DIR"
-  python -m pip install -r requirements.txt
+  python3.9 -m pip install -r requirements.txt
```

Comment:
I need to confirm, but I think `CODEZ` may use `python3.9` or `python3.10` and it may be dynamically decided based on x86 or m1 architecture.